apartment_application_service/settings.py

```diff
@@ -176,8 +176,35 @@
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "handlers": {"console": {"class": "logging.StreamHandler"}},
-    "loggers": {"django": {"handlers": ["console"], "level": env.str("LOG_LEVEL")}},
+    "formatters": {
+        "standard": {
+            "format": "%(asctime)s p%(process)d %(name)s %(levelname)s: %(message)s",
+            "datefmt": "%Y-%m-%dT%H:%M:%S.%03dZ",
+        }
+    },
+    "handlers": {
+        "console": {
+            "level": "INFO",
+            "class": "logging.StreamHandler",
+            "formatter": "standard",
+        }
+    },
+    "loggers": {
+        "": {
```

Comment:
Hmm.. do we need root logger configuration? Should we change this to `django`? In the servers, we have for some reason already configuration `DJANGO_LOG_LEVEL` so I'm thinking should we take it to use first? It needs to add envs as well which default value should be INFO like in `DEFAULT_LOGGING` settings. Otherwise if this not has any need we could also remove it completely for now.